### PR TITLE
fix: change list separator for the parameter packagePrefixes from ', …

### PR DESCRIPTION
**Pull request by iryabov**
…' to ';'

fix: change logging level validator to support logging level packages EPMDJ-10649

### CODE REVIEW

File: java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/Validators.kt [hunk 1]
```diff
@@ -48,6 +48,9 @@ fun ValidationBuilder<String>.isValidPackage(): Constraint<String> = addConstrai
     "must have a valid Java package delimited by a forward slash, e.g. 'com/example', but was '{value}'"
 ) { it.matches("^(?:[a-zA-Z_]\\w*(?:/[a-zA-Z_]\\w*)*)?$".toRegex()) }
 
+fun ValidationBuilder<String>.isValidLogLevel(): Constraint<String> = addConstraint(
+    "must have a valid logging level for a java package, e.g. 'com.example=INFO', but was '{value}'"
+) { it.matches("([\\w.]*=)?(TRACE|DEBUG|INFO|WARN|ERROR)".toRegex()) }
```
Comment (RomanDavlyatshin) [inline java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/Validators.kt:53]: fix + add tests for the following negative scenarios
```
"drill.=TRACE"
"com.epam..drill=TRACE"
".drill=TRACE"
```

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]
see https://github.com/Drill4J/java-agent/pull/126/files#r1340079535

**Review by RomanDavlyatshin — APPROVED** [review]
